### PR TITLE
gitless: 0.8.5 -> 0.8.6

### PR DESCRIPTION
**Pull request by cransom**
###### Motivation for this change
version bump.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/version-management/gitless/default.nix [hunk 1]
```diff
@@ -1,14 +1,14 @@
 { fetchFromGitHub, pythonPackages, stdenv }:
 
 pythonPackages.buildPythonApplication rec {
-  ver = "0.8.5";
+  ver = "0.8.6";
   name = "gitless-${ver}";
```
Comment (dotlambda) [inline pkgs/applications/version-management/gitless/default.nix]: Please specify `pname = "gitless"` istead.

File: pkgs/applications/version-management/gitless/default.nix [hunk 2]
```diff
@@ -1,14 +1,14 @@
 { fetchFromGitHub, pythonPackages, stdenv }:
 
 pythonPackages.buildPythonApplication rec {
-  ver = "0.8.5";
+  ver = "0.8.6";
```
Comment (dotlambda) [inline pkgs/applications/version-management/gitless/default.nix]: `version`

**Comment by cransom** [comment]
sorry for the lag time, fixed.

**Comment by cransom** [comment]
exists upstream already.

**Pull request closed without merging**